pyk4a/pyk4a.cpp

```diff
@@ -506,6 +507,9 @@ extern "C" {
             case K4A_IMAGE_FORMAT_COLOR_BGRA32:
                 pixel_size = (int)sizeof(uint32_t);
                 break;
+            case K4A_IMAGE_FORMAT_CUSTOM16:
+                pixel_size = static_cast<int32_t>(sizeof(int16_t));
```

Comment:
I don't think we use c++ syntax in this file. We tried to keep it clang style for simplicity. Can this be a problem when using a clang compiler? Maybe we should change all syntax to c++ (malloc to new, casts, etc.)